PiwikTracker/Device.swift

```diff
@@ -1,9 +1,9 @@
 import UIKit
 
-internal struct Device {
+public struct Device {
```

Comment:
I decided to remove the Device and Application struct from the SDK and moved it to the iOS example application. I think we should stick to our core tasks.